search/locality_finder.cpp

```diff
@@ -117,6 +118,17 @@ class LocalitiesLoader
   LocalityFinder::Holder & m_holder;
   unordered_set<uint32_t> & m_loadedIds;
 };
+
+int GetVillagesScale()
+{
+  feature::TypesHolder village;
+  village.Assign(classif().GetTypeByPath({"place", "village"}));
```

Comment:
Can we get types from IsVillageChecker here?

Reply:
done